lib/gssapi/krb5/8003.c

```diff
@@ -163,16 +163,85 @@ _gsskrb5_create_8003_checksum (
     return GSS_S_COMPLETE;
 }
 
+static krb5_error_code
+load_32_le(const void *in, uint32_t *out)
+{
+    krb5_error_code ret;
+    krb5_storage *sp;
+
+    sp = krb5_storage_from_readonly_mem(in, sizeof(uint32_t));
+    if (sp == NULL)
+	return ENOMEM;
+
+    krb5_storage_set_flags(sp, KRB5_STORAGE_BYTEORDER_LE);
+
+    ret = krb5_ret_uint32(sp, out);
+    krb5_storage_free(sp);
+
+    return ret;
+}
```

Comment:
This is very elegant but you could just use `_gss_mg_decode_le_uint32()` :)

Reply:
Thanks :)

Reply:
That worked.